lib/project_types/script/commands/connect.rb

```diff
@@ -5,6 +5,12 @@ class Connect < ShopifyCLI::Command::SubCommand
       prerequisite_task :ensure_authenticated
       prerequisite_task ensure_project_type: :script
 
+      recommend_node(
+        from: ::Script::Layers::Infrastructure::Languages::TypeScriptProjectCreator::MIN_NODE_VERSION,
+        to: ShopifyCLI::Constants::SupportedVersions::Node::TO
+      )
```

Comment:
@jacobsteves @lopert Please correct me if I'm wrong, but I don't believe we want to check node when we connect an app, since this command doesn't rely on node and the user could be using a language that doesn't require node (e.g. Rust, Wasm).